testbed/tests/widgets/test_webview.py

```diff
@@ -134,6 +136,24 @@ async def test_clear_url(widget, probe, on_load):
     )
 
 
+async def test_load_empty_url(widget, probe, on_load):
+    "An empty URL can be loaded asynchronously into the view"
+    await wait_for(
+        widget.load_url(None),
+        LOAD_TIMEOUT,
+    )
+
+    # DOM loads aren't instantaneous; wait for the URL to appear
+    await assert_content_change(
+        widget,
+        probe,
+        message="Page has been loaded",
+        url=None,
+        content=ANY,
```

Comment:
Should have the same assertions as `test_clear_url`.

```suggestion
        message="Page has been cleared",
        url=None,
        content="",
```